versions/1.17.4/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Let's move this to a `1.17` folder as to enforce a `/<major>.<minor>` policy where we, upon update to a new patch version would do so in the same folder.